client/src/components/products/Products.js

```diff
@@ -37,8 +15,7 @@ function Products({ userData, setUserData, productsData, setProduct }) {
             ? productsData.map((product) => (
                 <Card
                   key={product.id}
-                  product={product}
-                  setProduct={setProduct}
+                  product={product} //local mapped product variable that you shoud rename but will you?
```

Comment:
👁️

Reply:
😂 I was initally passing the products data i fetched from the api to this card, but i've moved that to context so i'll just remove the comment. It is not as confusing anymore.